effect-types/effect-types.Rmd

```diff
@@ -136,9 +136,9 @@ ITT<-coef(lm(Y~Z,data=samp))[2]
 ==
 What if you are interested in figuring out the effects of a treatment on those people who actually took up the treatment and not just those people that were administered the treatment? For example what is the effect of radio ads on voting behavior for those people that actually hear the ads?
 
-This turns out to be a hard problem (for more on this see this guide). The reasons for non-compliance with treatment can be thought of as an omitted variable. While the receipt of treatment is no-longer independent of potential outcomes, the assignment of treatment status is. As long as random assignment had some positive effect on the probability of receiving treatment, we can use it as an instrument to identify the effects of treatment on the sub-population of subjects who comply with treatment assignment.
+This turns out to be a hard problem (for more on this see [this guide](https://to_be_added)). The reasons for non-compliance with treatment can be thought of as an omitted variable. While the receipt of treatment is no-longer independent of potential outcomes, the assignment of treatment status is. As long as random assignment had some positive effect on the probability of receiving treatment, we can use it as an instrument to identify the effects of treatment on the sub-population of subjects who comply with treatment assignment.
```

Comment:
This should be a link to https://egap.org/resource/10-things-to-know-about-the-local-average-treatment-effect/

Reply:
Thanks for your comment.
I just pushed the new changes.